<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CondaPackagesWidget</class>
 <widget class="QWidget" name="CondaPackagesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QWidget" name="notAvailableWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="notAvailableLabel">
        <property name="text">
         <string>&lt;h2&gt;conda is not available&lt;/h2&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="recheckButton">
          <property name="toolTip">
           <string>Press to re-check the availability of conda</string>
          </property>
          <property name="text">
           <string>Re-Check</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="baseWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="environmentsComboBox"/>
        </item>
        <item>
         <widget class="E5ToolButton" name="condaMenuButton"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel"/>
      </item>
      <item>
       <widget class="QTreeWidget" name="packagesList">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>Package</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Installed Version</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Available Version</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="refreshButton">
          <property name="toolTip">
           <string>Press to refresh the lists</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upgradeButton">
          <property name="toolTip">
           <string>Press to upgrade the selected packages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upgradeAllButton">
          <property name="toolTip">
           <string>Press to upgrade all listed packages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="uninstallButton">
          <property name="toolTip">
           <string>Press to uninstall the selected package</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="searchToggleButton">
          <property name="toolTip">
           <string>Toggle to show or hide the search window</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="searchWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="searchEdit">
          <property name="placeholderText">
           <string>Enter search specification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="searchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Press to start the search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="searchOptionsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="patternButton">
             <property name="toolTip">
              <string>Search string is a pattern (default)</string>
             </property>
             <property name="text">
              <string>Search Pattern</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="fullNameButton">
             <property name="toolTip">
              <string>Search string is a full name</string>
             </property>
             <property name="text">
              <string>Full Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="packageSpecButton">
             <property name="toolTip">
              <string>Search string is a package specification</string>
             </property>
             <property name="text">
              <string>Package Specification</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Platform:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="platformComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Select the platform</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="searchResultList">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Package</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Version</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Build</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Platform</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="installButton">
          <property name="toolTip">
           <string>Press to install the selected package (by name or package specification)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="showDetailsButton">
          <property name="toolTip">
           <string>Press to show details for the selected entry</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ToolButton</class>
   <extends>QToolButton</extends>
   <header>E5Gui/E5ToolButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>environmentsComboBox</tabstop>
  <tabstop>condaMenuButton</tabstop>
  <tabstop>packagesList</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>upgradeButton</tabstop>
  <tabstop>upgradeAllButton</tabstop>
  <tabstop>uninstallButton</tabstop>
  <tabstop>searchToggleButton</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>patternButton</tabstop>
  <tabstop>fullNameButton</tabstop>
  <tabstop>packageSpecButton</tabstop>
  <tabstop>searchResultList</tabstop>
  <tabstop>installButton</tabstop>
  <tabstop>showDetailsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
